### ISSUE TITLE
Breaks neovim's :tcd command (tab-local cwds)

### ISSUE THREAD

**Issue by pianocomposer321** [hoob3rt/lualine.nvim#75]
Minimal init.lua:
```lua
lualine = require('lualine')
vim.cmd [[packadd packer.nvim]]
local plugins = require('packer').startup(function()
    use {'wbthomason/packer.nvim', opt = true}
    use {
        'hoob3rt/lualine.nvim',
        requires = 'kyazdani42/nvim-web-devicons',
    }
end)
lualine.status()
```

Steps to reproduce:
1. `nvim -u minimal-init.lua` (from home directory)
2. `:pwd`
3. `:tabnew`
4. `:tcd ~/.config/nvim`
5. `:pwd`
6. `gt`
7. `:pwd`

Expected result:
Prints "/home/user" first time `:pwd` is called, "/home/user/.config/nvim" the second time, and "/home/user" the third time.

Actual result:
Prints "/home/user" first time, "/home/user/.config/nvim" second _and third_ times.

The `:tcd` neovim command changes the cwd for only the current tab. But with this plugin installed, it changes it for all tabs.

**Comment by hoob3rt** [comment]
Wow that's something... I can't think of a reason why this would be the case, I will look into it :)

**Comment by pianocomposer321** [comment]
@hoob3rt I know, it surprised me too when I found out that this was the plugin causing the issue...I can't think of what would cause it with a statusline plugin. Thankfully, I'm not using tab-local cwd's much atm, I was just exploring them when I ran into this issue. So I'll continue using the plugin at this point, but definitely let me know if you find a fix or link in a patch.

**Comment by shadmansaleh** [comment]
I think I know what's causing it . It's the branch component . Before calling git cmd curdir is brefly changed to filedir . When I added that fix #37 I wasn't aware of this issue. 
Good thing is #68 fixes it as the branch component was redesined.
